Reorg Style Guide (#2956)
chef/chef-web-docs: content/style_html.md

+++

> title = "HTML and CSS Guide"

Reviewer: We should probably change this to Shortcodes since we're not documenting HTML or CSS.
```suggestion
title = "Shortcodes"
```

draft = false

gh_repo = "chef-web-docs"

[menu]
  [menu.overview]

>     title = "HTML & SCSS"

Reviewer: ```suggestion
    title = "Shortcodes"
```

    identifier = "overview/style/style_html Guide"
    parent = "overview/style"
    weight = 140
+++

## Content Reuse and Hugo Shortcodes

Chef docs uses [shortcodes](https://gohugo.io/content-management/shortcodes/) to maintain text that appears in more than one location and must be consistent in every location.

### Writing a shortcode

> All shortcode files are written in **Markdown** and stored in the `layouts/shortcodes` directory in the `chef-web-docs` repo.

Reviewer: ```suggestion
All shortcode files are written in **Markdown** and stored in the [`themes/docs-new/layouts/shortcodes`](https://github.com/chef/chef-web-docs/tree/master/themes/docs-new/layouts/shortcodes) directory in the [`chef-web-docs`](https://github.com/chef/chef-web-docs/) repository.
```
